pkg/errors/error_message.go

```diff
@@ -162,7 +162,8 @@ const (
 	TopicExistsOnPremErrorMsg            = `topic "%s" already exists for the Kafka cluster`
 	TopicExistsOnPremSuggestions         = "To list topics for the cluster, use `confluent kafka topic list --url <url>`."
 	FailedToProduceErrorMsg              = "failed to produce offset %d: %s\n"
-	MissingKeyErrorMsg                   = "missing key in message"
+	MissingKeyOrValueErrorMsg            = "missing key or value in message"
+	MissingOrMalformedKeyErrorMsg        = "missing or malformed key in message"
```

Comment:
Looks like this is only used in one package? Can we define it there instead of here?